EIPS/eip-9999.md

```diff
@@ -0,0 +1,285 @@
+---
+eip: 9999
+title: Enshrined Proposer Builder Separation
+description: Separates the ethereum block in consensus and execution parts, adds a mechanism for the consensus proposer to chose the execution proposer.
```

Comment:
Separates the Ethereum block in consensus and execution parts sounds like a structure change, but if you make it to 'Separate the **processing** of Ethereum block in consensus and execution' it is more powerful of a tag line IMO.

Reply:
Also chose => choose

Reply:
And execution proposer => builder